var/spack/repos/builtin/packages/xcrysden/package.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Xcrysden(MakefilePackage):
+    """XCrySDen is a crystalline and molecular structure visualisation program aiming at display of
+    isosurfaces and contours, which can be superimposed on crystalline structures and interactively
+    rotated and manipulated."""
+
+    homepage = "http://www.xcrysden.org/XCrySDen.html"
+    url = "http://www.xcrysden.org/download/xcrysden-1.6.2.tar.gz"
+
+    version("1.6.3-rc2", sha256="0565f55dfb67c73a824569bd2f02875f1e15c7214b86736fce4cacc3f5a189fe")
+    version("1.6.3-rc1", sha256="dac69eb6c37a64cb26e8e0fb378698786df61109ceea364f3878f0cbbe28c966")
+    version("1.6.2-cyg", sha256="609d7779fd14d5aac05332da012a34e3795a586a7d232621ce043ab1e6f103a4")
+    version("1.6.2", sha256="811736ee598bec1a5b427fd10e4e063a30dd7cadae96a43a50b36ce90a4f503f")
```

Comment:
Forgot to mention that packages typically flag the most recent stable version as preferred (i.e., add the `preferred=True` argument) instead of allowing release candidates take precedence.